ovos_audio/service.py

```diff
@@ -379,7 +381,9 @@ def handle_queue_audio(self, message):
             raise FileNotFoundError(f"{audio_file} does not exist")
         audio_ext = audio_ext or audio_file.split(".")[-1]
         listen = message.data.get("listen", False)
-        TTS.queue.put((audio_ext, str(audio_file), viseme, ident, listen))
+
+        sess_id = SessionManager.get(message).session_id
+        TTS.queue.put((audio_ext, str(audio_file), viseme, sess_id, listen, message))
```

Comment:
Note that `sess_id` is not unique here (neither was ident). a unique identifier for the `speak` event should be supported so playback end can be tracked per-`speak`